articles/machine-learning/how-to-prepare-datasets-for-automl-images.md

```diff
@@ -76,6 +80,35 @@ az ml data create -f [PATH_TO_YML_FILE] --workspace-name [YOUR_AZURE_WORKSPACE]
 [!Notebook-python[] (~/azureml-examples-main/sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb?name=upload-data)]
 ---
 
+If you already have your data present in Azure Blob Storage and want to create data asset out of it, you can do so by providing path to the location in Azure Blob Storage as shown below.
```

Comment:
Is it possible to just use the path on the azure blob storage in our jsonl files instead of creating a data asset.

Reply:
It appears creating data asset is not mandatory. Why do we have the section to create data asset in our notebooks ?
If we remove the use of data asset, then we should remove it from notebooks as well, before we remove it from docs. 
If we only skip reference to data asset while explaining direct data access from blob, then it changes rest of the code in notebook, which may create confusion